### PR TITLE
Rewrote the paragraph describing ways to access the NVDA menu, in user guide section "The NVDA Menu"

### LINKED ISSUE #14929: Clarify "applications key" and how to access the context menu in the user guide
### Is your feature request related to a problem? Please describe.

Following a query from a user about accessing the context menu on a keyboard without an applications key, it occured to me we should perhaps clarify it in the user guide.

### Describe the solution you'd like

In section 4.5: The NVDA menu, the applications key gets mentioned for the first time (in the context of explaining how to open the NVDA menu via the notification area shortcut as an alternate to using NVDA+n).  In that instruction, we note: "press the applications key located next to the right control key on most keyboards. "  It would be worth following this with a short sentence to say "On a keyboard without an applications key, press shift+f10".

### Describe alternatives you've considered

Although a purely Windows key (anyone without a screen reader running can use the aplications key or shift+f10 to open the context menu), it is worth noting both the key (as it currently is) and the alternative shift+f10 in the user guide because:
a) It is used in accessing the NVDA menu in that way, and
b) those on alternate keyboards (Apple users, or those with new compact keyboards) may not be familiar with the shift+f10 alternative when you don't have an applications key.
c) new users may not be familiar with the term "applications key".  I have also heard it called the "menu key" or the "context menu key".

### PR DESCRIPTION
**Pull request by XLTechie**
### Link to issue number:

Fixes #14929

### Summary of the issue:

Some users do not know that Shift+F10 is an alternative to the Applications key for accessing the Context menu.
As a good reason to provide this information in the user guide, make note of it in the first place which describes use of the Applications key: accessing the NVDA menu.

### Description of user facing changes

Altered the user guide section in question as follows:

* Since there were several ways listed to access the NVDA menu already, and #14929 asked to add another one: put all the methods in a list.
* Described the Shift+F10 alternative to the Applications key.
* Added code markers around all keys in that section, except "Windows logo key+B", which is written differently.
* Moved the right-click mouse option to last, since it is the least likely for a blind user.

### Description of development approach

Edited text. Saved text. Made commits. Did it some more.

### Testing strategy:

Have @Qchristensen read over the proposed changes. (Pending)

### Known issues with pull request:

None.

### Change log entries:
Changes
Listed all of the possible ways to reach the NVDA menu in the user guide, mentioning that Shift+F10 is an alternative to the Applications key for accessing the context menu.

### Code Review Checklist:

- [X] Pull Request description:
  - description is up to date
  - change log entries
- [X] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [X] API is compatible with existing add-ons.
- [X] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [X] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [X] Security precautions taken.

### CODE REVIEW

File: user_docs/en/userGuide.t2t [hunk 1]
```diff
@@ -482,10 +482,14 @@ The actual  commands will not execute while in input help mode.
...
 The NVDA menu allows you to control NVDA's settings, access help, save/revert your configuration, Modify speech dictionaries, access additional tools and exit NVDA.
 
-To get to the NVDA menu from anywhere in Windows while NVDA is running, press NVDA+n on the keyboard or perform a 2-finger double-tap on the touch screen.
-You can also get to the NVDA menu via the Windows system tray.
-Either right-click on the NVDA icon located in the system tray, or access the system tray by pressing the Windows logo key+B, DownArrow to the NVDA icon and press the applications key located next to the right control key on most keyboards.
-When the menu comes up, You can use the arrow keys to navigate the menu, and the enter key to activate an item.
+To get to the NVDA menu from anywhere in Windows while NVDA is running, you may do any of the following:
+- press ``NVDA+n`` on the keyboard.
+- Perform a 2-finger double-tap on the touch screen.
+- Access the system tray by pressing the Windows logo key+B, DownArrow to the NVDA icon, and press ``enter``.
```
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:488]: Fix up code formatting and simplify key names

```suggestion
- Access the system tray by pressing ``windows+b``, ``downArrow`` to the NVDA icon, and press ``enter``.
```

File: user_docs/en/userGuide.t2t [hunk 2]
```diff
@@ -482,10 +482,14 @@ The actual  commands will not execute while in input help mode.
...
+- Alternatively, access the system tray by pressing ``Windows+b``, DownArrow to the NVDA icon, and open the Context menu by pressing the ``applications`` key located next to the right control key on most keyboards. On a keyboard without an ``applications`` key, press ``shift+F10`` instead.
```
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:489]: Same here

```suggestion
- Alternatively, access the system tray by pressing ``windows+b``, ``downArrow`` to the NVDA icon, and open the Context menu by pressing the ``applications`` key located next to the right control key on most keyboards.
On a keyboard without an ``applications`` key, press ``shift+F10`` instead.
```

**Review by seanbudd — COMMENTED** [review]
Thanks @XLTechie

File: user_docs/en/userGuide.t2t [hunk 3]
```diff
@@ -485,8 +485,9 @@ The NVDA menu allows you to control NVDA's settings, access help, save/revert yo
 To get to the NVDA menu from anywhere in Windows while NVDA is running, you may do any of the following:
 - press ``NVDA+n`` on the keyboard.
 - Perform a 2-finger double-tap on the touch screen.
-- Access the system tray by pressing the Windows logo key+B, DownArrow to the NVDA icon, and press ``enter``.
-- Alternatively, access the system tray by pressing ``Windows+b``, DownArrow to the NVDA icon, and open the Context menu by pressing the ``applications`` key located next to the right control key on most keyboards. On a keyboard without an ``applications`` key, press ``shift+F10`` instead.
+- Access the system tray by pressing ``windows+b``, ``downArrow`` to the NVDA icon, and press ``enter``.
+- Alternatively, access the system tray by pressing ``windows+b``, ``downArrow`` to the NVDA icon, and open the Context menu by pressing the ``applications`` key located next to the right control key on most keyboards.
+On a keyboard without an ``applications`` key, press ``shift+F10`` instead.
```
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:490]: Hi, just fixing up some capitalization to be more consistent with the rest of the user guide.

```suggestion
- Access the system tray by pressing ``Windows+b``, ``downArrow`` to the NVDA icon, and press ``enter``.
- Alternatively, access the system tray by pressing ``Windows+b``, ``downArrow`` to the NVDA icon, and open the context menu by pressing the ``applications`` key located next to the right control key on most keyboards.
On a keyboard without an ``applications`` key, press ``shift+F10`` instead.
```

**Review by Qchristensen — APPROVED** [review]
Reads well.

**Pull request merged**